### ISSUE TITLE
supermium crashes after One-Core-API installed

### ISSUE THREAD

**Issue by cooxwh** [Skulltrail192/One-Core-API-Binaries#308]

**Comment by AmongusIsHere** [comment]
No, it says "The procedure entry point DiscardVirtualMemory could not be located in the dynamic link library progwrp.dll"
And I have sames problems with firefox, edge and chrome...

**Comment by TheLostMeme** [comment]
Having the same issue with Supermium 126 (latest build), hopefully the new OCA update fixes this crash on startup.

**Comment by Skulltrail192** [comment]
Fixed on 4.0.0,
